src/contexts/product-context.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect, useContext, createContext } from "react";
+import axios from "axios";
+
+const ProductContext = createContext();
+
+const ProductProvider = ({ children }) => {
+  const [loading, setLoading] = useState(false);
+  const [productList, setProductList] = useState([]);
+
+  const getProducts = async () => {
+    try {
+      setLoading(true);
+      const res = await axios.get("./api/products");
+      setProductList((_prev) => res.data.products);
+      setLoading(false);
+    } catch (error) {
+      console.log(error);
```

Comment:
I guess, loading should be set to false here too

Reply:
done